__tests__/app.test.js

```diff
@@ -76,3 +160,58 @@ describe('GET/api/articles/:article_id', () => {
   });
 });
 
+describe('GET/api/articles/:article_id/comments', () => {
+  it('GET:200 and should respond with an an array of comments for the given article_id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(11);
+        console.log(comments[0]);
+        console.log(comments);
+        expect(comments[1]).toMatchObject({
+          comment_id: 2,
+          votes: 14,
+          author: 'butter_bridge',
+          body: 'The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.',
+          article_id: 1,
+        })
+        comments.forEach((comment) => {
+          expect (comment.hasOwnProperty('comment_id')).toBe(true);
+          expect (comment.hasOwnProperty('votes')).toBe(true);
+          expect (comment.hasOwnProperty('created_at')).toBe(true);
+          expect (comment.hasOwnProperty('author')).toBe(true);
+          expect (comment.hasOwnProperty('body')).toBe(true);
+          expect (comment.hasOwnProperty('article_id')).toBe(true);
+        });
+      });
+    });
+    it('should return the comments sorted by date in descending order.', () => {
+      return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+          const { comments } = response.body;
+          expect(comments).toBeSortedBy('created_at',{
+            descending: true,
+          });
+        });
+    });
+  it('should throw 400 error if given a bad request', () => {
+    return request(app)
+    .get('/api/articles/banana')
```

Comment:
What endpoint is this for? 

- /api/articles/article_id
- /api/articles/article_id/comments 

are not the same and you have already tested these areas for the first endpoint of the two. You need to correct this so it represents to right endpoint.